testnet/stacks-node/src/nakamoto_node/sign_coordinator.rs

```diff
@@ -485,8 +482,7 @@ impl SignCoordinator {
             ));
         };
 
-        let start_ts = Instant::now();
-        while start_ts.elapsed() <= self.signing_round_timeout {
+        loop {
```

Comment:
No, I think the only right thing to do here is for the miner to try and wait forever until:

* enough signers reject the block
* enough signers sign the block (evidenced by either the arrival of enough signer signatures via StackerDB, or the signed Nakamoto block)
* the burnchain tip changes, and `config.miner.wait_on_interim_blocks` seconds have passed since it changed

Anything else invites the possibility that the miner produces arbitrarily many siblings at the same height, since network delays alone can ensure that the signers all see a bunch of block proposals in rapid succession (as perceived by each signer), which they would all sign and return signatures for (thereby creating siblings).  This is something I thought we wanted to avoid, per the discussion in #5132.

Reply:
Sure, that's a good idea.